### ISSUE TITLE
Cant get this working

### ISSUE THREAD

**Issue by gpopovic** [Gillespie59/eslint-plugin-angular#411]
I'm using eslint 3.2.2 and just can't get angular plugin working.

Eslint doesn't return any angular errors... :/

"eslint --debug" output

```
eslint:cli Running on files +0ms
  eslint:ignored-paths Looking for ignore file in /home/honeybooboo/go/src/bitbucket.org/efzg/psd/cmd/psd.js +31ms
  eslint:ignored-paths Could not find ignore file in cwd +3ms
  eslint:glob-util Creating list of files to process. +1ms
  eslint:cli-engine Processing /home/user/xxx/components/dashboard/dashboardController.js +8ms
  eslint:cli-engine Linting /home/user/xxx/components/dashboard/dashboardController.js +2ms
  eslint:config Constructing config for /home/user/xxx/components/dashboard/dashboardController.js +0ms
  eslint:config Using .eslintrc and package.json files +0ms
  eslint:config Loading /home/user/xxx/.eslintrc.yml +3ms
  eslint:config-file Loading YAML config file: /home/user/xxx/.eslintrc.yml +1ms
  eslint:config-file Loading eslint +109ms
  eslint:config-file Attempting to resolve eslint-config-eslint +1ms
  eslint:config-file Loading JS config file: /home/user/xxx/node_modules/eslint-config-eslint/index.js +0ms
  eslint:config-file Loading /home/user/xxx/node_modules/eslint/conf/eslint.json +187ms
  eslint:config-file Loading JSON config file: /home/user/xxx/node_modules/eslint/conf/eslint.json +0ms
  eslint:config Using /home/user/xxx/.eslintrc.yml +196ms
  eslint:config Merging command line environment settings +1ms
  eslint:config-ops Apply environment settings to config +0ms
  eslint:cli-engine Linting complete in: 612ms +95ms
```

.eslintrc.yml

```
extends: eslint
installedESLint: true
plugins:
  - angular
rules:
  no-use-before-define: 0
  no-underscore-dangle: 0
  no-constant-condition: 0
  angular/no-service-method: 0
  require-jsdoc: 0
```

when angular plugin is not installed.. eslint will output error.. meaning config is not the problem

**Comment by Gillespie59** [comment]
I think by default all rules are disabled. You should enabled them one by one.

**Comment by gpopovic** [comment]
@Gillespie59 yeah, it seems that you are correct. Any way to enable all rules ? 
It makes no sense to me to enable one by one.

**Comment by Gillespie59** [comment]
You can maybe use this shareable config : https://github.com/dustinspecker/eslint-config-angular, as explained in our README.md file

**Comment by Gillespie59** [comment]
Will be solved with #354
